standard/standard/clause_1_scope.adoc

```diff
@@ -1,16 +1,40 @@
 
-The GeoPose Standard is based on an implementation-neutral Logical Model (LM). This LM is a formalization of a Conceptual Model (CM). The CM consists of a linked set of terms and definitons, defining a domain odf discourse for the various geometrric, geographic, and physical concepts related to GeoPoses. The LM formalizes the relationships between the implementable parts and aspects of the CM. The LM further establishes the structure and relationships between GeoPose components and also between GeoPoses data objects themselves in composite structures. 
+The GeoPose Standard is based on an implementation-neutral Logical Model (LM). This LM
+is a formalization of a Conceptual Model (CM). The CM consists of a linked set of terms
+and definitons, defining a domain odf discourse for the various geometrric, geographic,
+and physical concepts related to GeoPoses. The LM formalizes the relationships between
+the implementable parts and aspects of the CM. The LM further establishes the structure
+and relationships between GeoPose components and also between GeoPoses data objects
+themselves in composite structures.
 
-Note that the concrete GeoPose data objects defined by this standard correspond to only certain classes and properties of the LM. These classes and properties are identified as implementation-neutral Structural Data Units (SDUs). SDUs are  aliases for the implementable elements of the LM. SDUs are grouped to define the implementation-neutral form of the GeoPose Standardization Targets: the specific implementation that the Standard addresses. For each Standardization Target, each implementation technology will have the definition of the encoding or serialization specified in a manner appropriate to that technology.
+Note that the concrete GeoPose data objects defined by this standard correspond to only
+certain classes and properties of the LM. These classes and properties are identified
+as implementation-neutral Structural Data Units (SDUs). SDUs are  aliases for the
+implementable elements of the LM. SDUs are grouped to define the implementation-neutral
+form of the GeoPose Standardization Targets: the specific implementation that the
+Standard addresses. For each Standardization Target, each implementation technology
+will have the definition of the encoding or serialization specified in a manner
+appropriate to that technology.
 
-GeoPose Version 1.0 defines only one of many possible encoding methods for implementation of any or all of the eight STandardization Targets: JavaScript Object Notation (JSON). Each Standardization Target has a JSON-Schema:2019-9 definition. Most of the GeoPose standardization requirements are that concrete JSON GeoPose data objects shall conform to the corresponding JSON-Schema definition. The individual elements identified in the encoding specifications are SDUs that refer to one or more classes or attributes of the LM.
+GeoPose Version 1.0 defines only one of many possible encoding methods for
```

Comment:
Let's not re-format the text when we don't need to.

Reply:
I mean not to re-flow the text (do not break the paragraph into separate lines).